hypothesis-python/tests/array_api/test_argument_validation.py

```diff
@@ -61,6 +69,8 @@ def e(name, **kwargs):
         e("unsigned_integer_dtypes", sizes=(3,)),
         e("floating_dtypes", sizes=()),
         e("floating_dtypes", sizes=(3,)),
+        e("complex_dtypes", _min_version="draft", sizes=()),
+        e("complex_dtypes", _min_version="draft", sizes=(3,)),
```

Comment:
(see comment about marker above)  Maybe we actually just want a `MIN_VER_FOR_COMPLEX = "draft"` constant?